test/modules/linear_boltzmann_solvers/transport_keigen/keigenvalue_transport_3d_2g_exo_hex.lua

```diff
@@ -0,0 +1,84 @@
+-- 3D 2G Infinite Medium Hex import test. Imports EXODUSII.
+-- Uses KEigenvalue::Solver with Power Iteration
+-- Test: Final k-eigenvalue: 0.9293376
+
+-- Set and check number of processors
+num_procs = 1
+if check_num_procs == nil and number_of_processes ~= num_procs then
+  log.Log(
+    LOG_0ERROR,
+    "Incorrect amount of processors. "
+      .. "Expected "
+      .. tostring(num_procs)
+      .. ". Pass check_num_procs=false to override if possible."
+  )
+  os.exit(false)
+end
+
+-- Setup mesh
+meshgen1 = mesh.MeshGenerator.Create({
+  inputs = {
+    mesh.FromFileMeshGenerator.Create({
+      filename = "../../../assets/mesh/fuel_hex.e",
+    }),
+  },
+  partitioner = mesh.PETScGraphPartitioner.Create({ type = "parmetis" }),
```

Comment:
Rethink this line:
- If you want to run on 1 mpi, then you should not need this line. 
- If you want to have a parallel run, then change the restriction above.